merger

```diff
@@ -0,0 +1,20 @@
+def combine_chances(textual_chance: float, numerical_chance: float):
+    # 1. Combine the two numbers into an average value
+    average_value = (textual_chance + numerical_chance) / 2
+
+    # 2. Display the median of the two values
+    median_value = sorted([textual_chance, numerical_chance])[1]  # Since there are only two values, the second one is the median
+
+    # 3. Show the maximum value
+    max_value = max(textual_chance, numerical_chance)
+
+    # 4. Show the minimum value
+    min_value = min(textual_chance, numerical_chance)
+
+    # Display results
+    print(f"Average Value: {average_value:.2f}%")
+    print(f"Median Value: {median_value:.2f}%")
+    print(f"Maximum Value: {max_value:.2f}%")
+    print(f"Minimum Value: {min_value:.2f}%")
```

Comment:
I suggest to output dict of values, e.g.
```
return {
    "Average Value": average_value,
...
}
```

and then make another function `print_results(data: dict)`
which make this nice print

Final call could be
```
print_results(combine_chances(textual_chance, numerical_chance))
```